### PR TITLE
Add Design Principles

### PR DESCRIPTION
**Pull request by robinstickel**
**[Awesome Design Principles](https://github.com/robinstickel/awesome-design-principles)**

A curated list of good design and experience principles.

Design Principles are a tool for creating better and more consistent designs and experiences. They are high level rules that guide detailed design decisions. They are often written down within bigger style guides, pattern libraries or design systems.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -553,6 +553,7 @@
 - [Homematic](https://github.com/hobbyquaker/awesome-homematic) - Smart home devices.
 - [Ledger](https://github.com/sfischer13/awesome-ledger) - Double-entry accounting on the command-line.
 - [Uncopyright](https://github.com/johnjago/awesome-uncopyright) - Public domain works.
+- [Design Principles](https://github.com/robinstickel/awesome-design-principles) - Design and user experience principles.
```
Comment (sindresorhus) [inline readme.md]: The description is just partly repeating the title. I would use this instead:

> Create better and more consistent designs and experiences.

**Comment by sindresorhus** [comment]
> Each link description starts with an uppercase character and **ends with a period.**
Example: `- [AVA](…) - JavaScript test runner.`

Link and description should also be separated by a dash.

**Comment by robinstickel** [comment]
Hey @sindresorhus thanks for your feedback. Indeed, your description is way better so I changed it.

**Comment by sindresorhus** [comment]
Thanks :)

**Pull request merged**